libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.m

```diff
@@ -45,6 +45,13 @@
 // app when it is re-opened.
 static NSString * const kAppSettingsAccountLogout = @"account_logout_pref";
 
+// Random id to uniquely identify user
+static NSString* uid;
```

Comment:
It's always good to initialize static objects to `nil`, for some weird edge case stuff.